src/sentry/static/sentry/app/views/settings/organization/navigationConfiguration.tsx

```diff
@@ -74,6 +74,10 @@ const organizationNavigation: NavigationSection[] = [
           'Manage organization-level integrations, including: Slack, Github, Bitbucket, Jira, and Azure DevOps'
         ),
         id: 'integrations',
+        analyticsParams: {
```

Comment:
👎 for having unique event keys for each of the navigation items, IMO what we should do is have a single `eventKey` / `eventName` for all sidebar navigations.

I'm fine with having parameters specific to each nav item, though I don't know what they would be.

Reply:
As discussed in person, I will use another approach